tests/apis/parameters.spec.ts

```diff
@@ -51,17 +50,17 @@ describe("getParameter API", () => {
     context("@sap/plugin loaded", () => {

```

Comment:
context annotation name is repeated several times ...

Reply:
done